### PR TITLE
feat: added login and register component

### PR DESCRIPTION
**Pull request by mansi2011**
feat: added login and register component

login

### CODE REVIEW

File: Component/Account/account.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    color: var(--white-font-color);
+    background: var(--btn-background);
+    border-radius: 8rem;
+}
+
+.btn-register{
+    width: 50%;
+    border: 1px solid var(--btn-background);
+    cursor: pointer;
+    margin: 1rem 0;
```
Comment (tewarimohit) [inline Component/Account/account.css:56]: try to declare it globally..

File: Component/Account/account.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    padding: 0 1rem;
+    border: 1px solid var(--footer-font-color);
+    border-radius: 8rem;
+}
+
+.form-btn{
+    width: 100%;
+    border: none;
+    cursor: pointer;
+    margin: 1rem 0;
```
Comment (tewarimohit) [inline Component/Account/account.css:45]: try to declare common properties as global..

File: Component/Account/account.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    padding: 1rem 2rem;
```
Comment (tewarimohit) [inline Component/Account/account.css:46]: same with padding

File: Component/Account/account.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    background: var(--white-font-color);
+}
+
+#register-form{
+    display: none;
+}
+
+#display-register{
+    display: none;
+}
```
Comment (tewarimohit) [inline Component/Account/account.css:69]: you can make display:none; as class and can use it in different places..

**Review by tewarimohit — COMMENTED** [review]
Just go through comments.... otherwise Code looks good to me..

File: app.js [hunk 5]
```diff
@@ -8,4 +8,24 @@ if(menuItems.style.maxHeight == "0px"){
...
\ No newline at end of file
+}
+
+
+var loginForm = document.getElementById("login-form");
+var registerForm = document.getElementById("register-form");
+var registerBtn = document.getElementById("register");
+var accountExist = document.getElementById("account-boolean");
+var displayLogin = document.getElementById("display-login");
+var displayRegister = document.getElementById("display-register")
```
Comment (parthgabani777) [inline app.js:19]: you can use const here.

File: Component/Account/account.css (hunk 4 above)
Comment (mansi2011) [reply Component/Account/account.css:69]: okay

File: app.js (hunk 5 above)
Comment (mansi2011) [reply app.js:19]: okay

**Pull request merged**